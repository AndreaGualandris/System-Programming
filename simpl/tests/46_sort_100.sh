#!/bin/bash
exec "$1" tests/SORT <<EOF
1365
1332
755
2478
1894
677
1883
2265
166
2032
1464
331
1622
269
673
767
1091
1803
1803
2402
1350
2115
661
289
1381
1320
549
411
1866
1576
1851
215
727
1296
2380
1741
1624
1236
280
2227
1608
2382
689
709
1619
2306
74
2102
1156
11
2493
658
2291
878
61
2167
1454
2065
2249
170
1966
457
1348
2165
1884
1879
996
296
793
2091
2245
1839
2422
1256
2492
698
1862
817
718
228
431
473
108
1855
1679
2460
1339
467
884
1414
680
2390
1584
88
408
349
938
2412
1910
2007
EOF
