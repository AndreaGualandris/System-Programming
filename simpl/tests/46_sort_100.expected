11
61
74
88
108
166
170
215
228
269
280
289
296
331
349
408
411
431
457
467
473
549
658
661
673
677
680
689
698
709
718
727
755
767
793
817
878
884
938
996
1091
1156
1236
1256
1296
1320
1332
1339
1348
1350
1365
1381
1414
1454
1464
1576
1584
1608
1619
1622
1624
1679
1741
1803
1803
1839
1851
1855
1862
1866
1879
1883
1884
1894
1910
1966
2007
2032
2065
2091
2102
2115
2165
2167
2227
2245
2249
2265
2291
2306
2380
2382
2390
2402
2412
2422
2460
2478
2492
2493
